extensions/mssql/src/objectExplorerNodeProvider/hdfsCommands.ts

```diff
@@ -71,8 +71,8 @@ export class UploadFilesCommand extends ProgressCommand {
 		try {
 			let folderNode = await getNode<FolderNode>(context, this.appContext);
 			const allFilesFilter = localize('allFiles', "All Files");
-			let filter = {};
-			filter[allFilesFilter] = '*';
+			let filter: { [key: string]: string[] } = {};
+			filter[allFilesFilter] = ['*'];
```

Comment:
Functional change here. This is defined by the vscode apis. I suspect the only reason it was working before is vscode had some back compat code in place to catch this.